src/utils/display-text/format.ts

```diff
@@ -109,5 +109,13 @@ export function calculateProfit(
   return isLong ? profitLong : profitShort;
 }
 
-export const stringToFixedPrecision = (value: string, precision: number) =>
-  parseFloat(value).toFixed(precision);
+export const countDecimals = value => {
```

Comment:
You don't need to export this function since the only usage is in the same file (and I don't think we would need it elsewhere). 

Also, rename it to `decimalPartLength` please.

Reply:
Why you didn't address this comment? Let me know if you have another opinion on it but I think those are valid arguments.